### ISSUE TITLE
should return 403 response-code for METHOD_NOT_ALLOWED

### ISSUE THREAD

**Issue by IswaryaaS** [openBackhaul/ApplicationPattern#647]
- For accessing any resource in OAM layer, we are communicating with AdministratorAdministration application. 
- AA will respond with 200 OK with corresponding response in response-body.
- In AA, we have "allowed-methods" parameters to check if the corresponding user requesting to access an resource has permission to certain actions. example: A user with GET only access could not "PUT". In this case, AA will return "METHOD_NOT_ALLOWED" message in response-body.

Based on above example, a user is authenticated by AA, but not authorized to access that particular resource. For such cases, the response-code would not be simply 401 unauthorized. But it could be 403 forbidden. 

Please let me know your views.

**Comment by PrathibaJee** [comment]
Yes , incase of unauthorized access , the http response code should be 403.
**Reference :** https://tools.ietf.org/html/rfc7231#section-6.5.3

**Next step :** 
In applicationPattern/services/AuthorizingService.js , isAuthorized method needs to include the additional logics, 

if "oam-request-is-approved" from AA is false
then 
if "reason-of-objection" is 'METHOD_NOT_ALLOWED'
then
response code should be 403
